src/java.base/share/classes/java/io/Console.java

```diff
@@ -82,30 +80,35 @@
  * manually zero the returned character array after processing to minimize the
  * lifetime of sensitive data in memory.
  *
- * <blockquote><pre>{@code
+ * {@snippet lang=java :
  * Console cons;
  * char[] passwd;
  * if ((cons = System.console()) != null &&
  *     (passwd = cons.readPassword("[%s]", "Password:")) != null) {
  *     ...
  *     java.util.Arrays.fill(passwd, ' ');
  * }
- * }</pre></blockquote>
+ * }
  *
  * @author  Xueming Shen
  * @since   1.6
+ * @sealedGraph
  */
+public sealed class Console implements Flushable permits ConsoleImpl, ProxyingConsole {
+    /**
+     * Package private no-arg constructor.
+     */
+    Console() {}
 
-public class Console implements Flushable
-{
    /**
     * Retrieves the unique {@link java.io.PrintWriter PrintWriter} object
     * associated with this console.
     *
     * @return  The printwriter associated with this console
     */
     public PrintWriter writer() {
-        return pw;
+        throw new UnsupportedOperationException(
+                "Console class itself does not provide implementation");
```

Comment:
Minor comment is the same "throw new UOE(message)" is repeated in every method. You could add a static method to return it, then it would reduce to "throw newUnsupportedOperationException();".

Also looks like the method descriptions on all these methods starts at position 3. Pre-dates your change but since every method is changed then maybe we should fix that too, up to you.

Reply:
Thanks, Alan. Addressed both.